packages/projection/src/operators/certificates/withAdaHandle.ts

```diff
@@ -0,0 +1,12 @@
+import { WithCertificates } from './withCertificates';
+import { unifiedProjectorOperator } from '../utils';
+
+export interface WithAdaHandle {
+  address: string;
+  handle: string;
+}
+
+export const withAdaHandle = unifiedProjectorOperator<WithCertificates, WithAdaHandle>((evt) =>
```

Comment:
The 1st generic argument is for dependencies. This operator will probably not need any?
```suggestion
export const withAdaHandle = unifiedProjectorOperator<{}, WithAdaHandle>((evt) =>
```